website/docs/guides/unleash-edge-quickstart.md

```diff
@@ -39,7 +39,7 @@ It exposes the same HTTP interface as the main [Unleash API](/api-overview) but
 
 Here's what you need before getting started:
 
-1. An [Unleash instance](https://docs.getunleash.io/quickstart) running locally or remotely (version `4.15` or later)
+1. An [Unleash instance](https://docs.getunleash.io/get-started/quickstart) running locally or remotely (version `4.15` or later)
```

Comment:
should we use absolute links (without the full hostname) like everywhere else? sorry this was my quickstart guide :)

Reply:
fixed in 85b8c9999c35b89b9567e371a143b5fe67491842